CHANGELOG.md

```diff
@@ -1,6 +1,92 @@
+
+### `Prepare` Statement
+
+* Prohibit adding the `Prepare` statement containing subqueries to the query plan cache to guarantee the query plan is correct when different user variables are input [#8064](https://github.com/pingcap/tidb/pull/8064)
+* Optimize the query plan cache to guarantee the plan can be cached when the statement contains non-deterministic functions [#8105](https://github.com/pingcap/tidb/pull/8105)
+* Optimize the query plan cache to guarantee the query plan of `DELETE`/`UPDATE`/`INSERT` can be cached [#8107](https://github.com/pingcap/tidb/pull/8107)
+* Optimize the query plan cache to remove the corresponding plan when executing the `DEALLOCATE` statement [#8332](https://github.com/pingcap/tidb/pull/8332)
+* Optimize the query plan cache to avoid the TiDB OOM issue caused by caching too many plans by limiting the memory usage [#8339](https://github.com/pingcap/tidb/pull/8339)
+* Optimize the `Prepare` statement to support using the `?` placeholder in the `ORDER BY`/`GROUP BY`/`LIMIT` clause [#8206](https://github.com/pingcap/tidb/pull/8206)
+
+### Privilege Management
+
+* Add the privilege check for the `ANALYZE` statement [#8486](https://github.com/pingcap/tidb/pull/8486)
+* Add the privilege check for the `USE` statement [#8414](https://github.com/pingcap/tidb/pull/8418) 
+* Add the privilege check for the `SET GLOBAL` statement [#8837](https://github.com/pingcap/tidb/pull/8837)
+* Add the privilege check for the `SHOW PROCESSLIST` statement [#7858](https://github.com/pingcap/tidb/pull/7858)
+
+### Server
+
+* Support the `Trace` feature [#9029](https://github.com/pingcap/tidb/pull/9029)
+* Support the plugin framework [#8788](https://github.com/pingcap/tidb/pull/8788)
+* Support using `unix_socket` and TCP simultaneously to connect to the database [#8836](https://github.com/pingcap/tidb/pull/8836)
+* Support the `interactive_timeout` system variable  [#8573](https://github.com/pingcap/tidb/pull/8573)
+* Support the `wait_timeout` system variable [#8346](https://github.com/pingcap/tidb/pull/8346)
+* Support splitting a transaction into multiple transactions based on the number of statements using the `tidb_batch_commit` variable [#8293](https://github.com/pingcap/tidb/pull/8293)
+* Support using the `ADMIN SHOW SLOW` statement to check slow logs [#7785](https://github.com/pingcap/tidb/pull/7785)
+
+### Compatibility
+
+* Support the `ALLOW_INVALID_DATES` SQL mode [#9027](https://github.com/pingcap/tidb/pull/9027)
+* Improve `LoadData` fault-tolerance for the CSV file [#9005](https://github.com/pingcap/tidb/pull/9005)
+* Support the MySQL 320 handshake protocol [#8812](https://github.com/pingcap/tidb/pull/8812)
+* Support stating the unsigned bigint column as the auto-increment column [#8181](https://github.com/pingcap/tidb/pull/8181)
+* Support the `SHOW CREATE USER` syntax [#8970](https://github.com/pingcap/tidb/pull/8970)
```

Comment:
As I understand, this is only parser support for `SHOW CREATE USER` syntax.  I think it is not relevant for release notes until the feature has been implemented?

Reply:
What do you think @shenli ?